notebooks/MAST/m_dwarf_tess/m_dwarf_flares_tess.ipynb

```diff
@@ -0,0 +1,1285 @@
+{
```

Comment:
<p>This is great motivation for this section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>